requirements.txt

```diff
@@ -8,11 +8,11 @@ ecdsa==0.13
 gmpy==1.17
 jsonrpc==1.2
 jsonrpclib==0.1.7
-https://github.com/lbryio/lbryum/tarball/master/#egg=lbryum
+https://github.com/lbryio/lbryum/tarball/update-protobuf/#egg=lbryum
```

Comment:
should this be master? or should that be a separate commit after lbryum gets merged?